tests/wpt-harness/expectations/fetch/api/basic/request-upload.any.js.json

```diff
@@ -48,7 +48,7 @@
     "status": "FAIL"
   },
   "Fetch with POST with text body on 421 response should be retried once on new connection.": {
-    "status": "PASS"
+    "status": "FAIL"
```

Comment:
Do you know why this test is failing now?

Reply:
The more surprising result was that it ever reported as passing, our fetch implementation does not include retry logic - which is what this test is attempting to exercise

Reply:
Quite mysterious!